src/fu-self-test.c

```diff
@@ -1754,6 +1755,19 @@ fu_common_firmware_builder_func (void)
 		g_test_skip ("no bwrap in path, so skipping");
 		return;
 	}
+	if (!g_file_test ("/proc/self/ns/user", G_FILE_TEST_IS_SYMLINK)) {
```

Comment:
I kinda feel this needs to be split out into another function in this file -- perhaps we can even use it in the daemon?

Reply:
Split out and added to other places builder is called